samples/zigbee/light_switch/src/main.c

```diff
@@ -331,10 +339,15 @@ static void find_light_bulb(zb_bufid_t bufid)
 	/* Set 0xFFFF to reset short address in order to parse
 	 * only one response.
 	 */
-	device_ctx.bulb_params.short_addr = 0xFFFF;
+	bulb_ctx.short_addr = 0xFFFF;
 	(void)zb_zdo_match_desc_req(bufid, find_light_bulb_cb);
 }
 
+static void find_light_bulb_timeout_alarm(struct k_timer *timer)
+{
+	zigbee_schedule_callback(find_light_bulb_timeout, bulb_ctx.bufid_tmo);
```

Comment:
As above.

Reply:
The same answer.

Reply:
I removed find_light_bulb_timeout_alarm function.